### ISSUE TITLE
Bug - Accessibility - Deleting an element in a list causes focus to jump to "skip to main content" link

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/TalentCloud#5461]
# Description
We should more gracefully handle focus for delete buttons which remove elements from a list. The problem is that if the delete button removes itself from the DOM when the item is deleted, focus will automatically jump to the first focusable element in the page (the skip link button).

**Question**: What is the ideal behaviour here? And where are all the places we need to implement it? 
- Focus should be moved to the next focusable element.
- At a minimum, we need to implement this on the Experience and Skills pages of the Application and Profile. Jobs list for Manager.

**Comment by tristan-orourke** [comment]
@bitliege @gggrant Could use your feedback on desired behaviour.
